frontend/scss/_common.scss

```diff
@@ -98,3 +101,7 @@ img {
   @extend .section;
   padding: 2rem 0;
 }
+
+.section.mission.blue-section.style-scope.home-page {
+  display: none;
```

Comment:
What is the point of hiding this?

Reply:
class name has been removed from style sheet